partner_phone_secondary/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Make it https:

Personally I would like to have this as a header in all files, certainly also in the manifest file.

Reply:
done